examples/nextjs-comments-audio/src/components/ThreadsTimeline.tsx

```diff
@@ -0,0 +1,111 @@
+"use client";
+
+import { Link } from "@/components/Link";
+import { Mention } from "@/components/Mention";
+import { ThreadMetadata, useThreads, useUser } from "@/liveblocks.config";
+import { useSkipTo } from "@/utils";
+import { ThreadData } from "@liveblocks/core";
+import { ClientSideSuspense } from "@liveblocks/react";
+import { Comment } from "@liveblocks/react-comments/primitives";
+import { CSSProperties, useCallback, useState } from "react";
+import { ErrorBoundary } from "react-error-boundary";
+
+export function ThreadsTimeline() {
+  return (
+    // @ts-ignore
```

Comment:
What's the TS error here?

Reply:
This is a bit of a rabbit hole! If I recall correctly, the only way to fix this is to downgrade `@types/react`, so I'd prefer leave leave in the `ts-ignore`